distribution/docs/src/main/resources/content/_tables/catalog.ui.config.adoc

```diff
@@ -252,30 +324,134 @@ TMS example (3d map support only): `{ "name": "Example TMS", "show": true, "type
 |Maximum Endpoint Upload Size
 |maximumUploadSize
 |Integer
-|The maximum size of data that the Intrigue REST endpoints will receive. Please note this is not related to the product upload size, this only refers to the json data sent and received from the Intrigue endpoints.
+|The maximum size (in bytes) to allow per client when receiving a POST/PATCH/PUT. Note: This does not affect product upload size, just the maximum size allowed for calls from Intrigue.
 |1048576
-|false
+|true
 
 |Map Home
 |mapHome
 |String
-|Specifies the default home view for the map by bounding box.  The format is: 'West, South, East, North' where North, East, South, and West are coordinates in degrees.  An example is: '-124, 60, -100, 40'.
+|Specifies the default home view for the map by bounding box. The format is: "West, South, East, North" where North, East, South, and West are coordinates in degrees. An example is: `-124, 60, -100, 40`.
 |
 |false
 
 |UI Branding Name
 |uiName
 |String
-|Specifies a custom UI branding name in the UI. This defaults to 'Intrigue'.
+|Specifies a custom UI branding name in the UI.
 |Intrigue
 |true
 
+|Show Relevance Scores
+|showRelevanceScores
+|Boolean
+|Toggle the display of relevance scores of search results.
+|false
+|false
+
+|Relevance Score Precision
+|relevancePrecision
+|Integer
+|Set the number of digits to display in for each relevance score. The default is `5` (i.e. 12.345).
+|5
+|false
+
 |Show Logo in Title Bar
 |showLogo
 |Boolean
-|Toggles the visibility of the logo in the title bar.
+|Toggles the visibility of the logo in the menu bar.
 |false
 |false
 
-|===
+|Theme
+|theme
+|String
+|Specifies the default theme. `Custom` consists of the colors below.
+|Dark
+|true
+
+|Primary Color
+|String
```

Comment:
Looks like something got mixed up here. I think the String line needs to be swapped with the customPrimaryColor.